Userland/DevTools/HackStudio/Language.cpp

```diff
@@ -13,14 +13,18 @@ Language language_from_file_extension(const String& extension)
     VERIFY(!extension.starts_with("."));
     if (extension == "cpp" || extension == "h")
         return Language::Cpp;
-    else if (extension == "js")
+    else if (extension == "js" || extension == "json")
         return Language::JavaScript;
+    else if (extension == "html" || extension == "htm")
+        return Language::HTML;
     else if (extension == "gml")
         return Language::GML;
     else if (extension == "ini")
         return Language::Ini;
-    else if (extension == "sh")
+    else if (extension == "sh" || extension == "bash")
```

Comment:
Who uses `foo.bash` as a shell script...?

Reply:
https://github.com/search?q=extension%3Abash - "530,045 code results".

Reply:
Compared to https://github.com/search?q=extension%3Ash&type= "29 million" it's...like 2%?
That's pretty low, but okay I guess